docker-entrypoint.sh

```diff
@@ -14,25 +14,19 @@
 
 # Help message that is printed if the container is not startes as recommended.
 HELP_MESSAGE='
-The recommended way to run a container with this image is as follows. Run in a fresh directory. Add `-p <outside port>:<inside port>` if you want to expose ports. Inside the container, the `qlever` command-line tool is available, as well as the QLever binaries (which you need not call directly, they are called by the various `qlever` commands).
+The recommended way to use this image is with the `qlever` command-line tool, which can be installed with `pip install qlever`, `pipx install qlever`, or `uv tool install qlever`. Commands like `qlever index` or `qlever start` use the QLever Docker image by default (and will pull the right image if necessary).
```

Comment:
> will pull the right image if necessary

For some users it might be confusing that if a `qlever:latest` image already exists locally (regardless of age) no new image is pulled and they have to pull it manually if they want a more up-to-date version.